Go/src/mqsamputils/connection.go

```diff
@@ -65,20 +71,19 @@ func CreateConnection(index int) (ibmmq.MQQueueManager, error) {
 
 	// Allocate the MQCNO structure needed for the CONNX call.
 	cno := ibmmq.NewMQCNO()
-  env := getEndPoint(index)
+    env := getEndPoint(index)
```

Comment:
In my editor this line is aligned, whereas here it is not.

Reply:
I think there is a mix of spaces and tabs, you will need to run a code beautify.